Minecraft/launchClient.bat

```diff
@@ -84,6 +84,7 @@ if "!-replaceable!"=="true" (
 
 :launchLoop
 REM finally run Minecraft:
+REM cd %~dp0
```

Comment:
remove as commented out

Reply:
Actually I accidentally put it as a comment. I want it to be there.

Reply:
So you do want to change to the dir that the script is in? We call launchClient.bat from other scripts so would need to check that the cd (without the REM) does not break these.

Reply:
I've dropped it from this patch. Let users do `cd %MALMO_MINECRAFT_ROOT%` and then call launchClient instead of invoking it with absolute path.